src/lib/components/Modal/__tests__/ModalContent.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  render, within,
+} from '@testing-library/react';
+import { ModalContent } from '../ModalContent';
+
+const mandatoryProps = {
+  children: <div>content text</div>,
+};
+
+describe('rendering', () => {
+  it.each([
+    [
+      {
+        children: (
+          <>
+            <div>content text 1</div>
+            <div>content text 2</div>
+          </>
+        ),
+      },
+      (rootElement) => {
+        expect(within(rootElement).getByText('content text 1'));
+        expect(within(rootElement).getByText('content text 2'));
+      },
+    ],
+    [
+      {
+        id: 'id',
```

Comment:
There is a reusable `idPropTest.js` if I'm not mistaken, can we use it where applicable?